demisto_sdk/commands/postman_codegen/postman_codegen.py

```diff
@@ -447,3 +450,23 @@ def duplicate_requests_check(commands_names_dict: dict) -> None:
     assert len(duplicates_list) == 0, f'There are requests with non-unique names: {duplicates_list}.\n' \
                                       f'You should give a unique name to each request.\n' \
                                       f'Names are case-insensitive and whitespaces are ignored.'
+
+
+def find_shared_args_path(flattened_json: dict) -> dict:
+    names_dict: dict = defaultdict(list)
```

Comment:
names_dict doesn't describe this object well enough, maybe ` arg_name_to_path_dict ` would describe it better.